.buildkite/x-pack/pipeline.xpack.winlogbeat.yml

```diff
@@ -18,6 +18,8 @@ env:
   RACE_DETECTOR: "true"
   TEST_COVERAGE: "true"
 
+  BEAT_PATH: "x-pack/winlogbeat"
```

Comment:
x-pack.winlogbeat didn't need the BEAT_PATH in the main branch? https://github.com/elastic/beats/blob/main/.buildkite/x-pack/pipeline.xpack.winlogbeat.yml

Reply:
You are right, it shouldn't be there. For x-pack/winlogbeat in Jenkinsfile within 7 steps running `mage build unitTest`, only 1 has `withModule` in both main and 7.17 - which seems to be an inconsistency. I have mentioned it [here](https://github.com/elastic/ingest-dev/issues/3355#issuecomment-2252584507).
Will remove it.